Hangman/Hangman/main.swift

```diff
@@ -8,5 +8,168 @@
+// option command left
+// to unfold is in editor
+
+    // Instructions
+print("WELCOME tooo Hangman, a game that is much more tedious and stressfull than your middle school spelling bee because one wrong move and someone's gonna end up dead(figruatively of course).")
+print()
+
+print("Please be mindful that you only have 6 times to miss a letter before you are hanged... Also we only want letters NOT numbers. ")
+
+print()
+
+print("Your word has been chosen... Now it time for you to guess the word.. good luck.")
+print()
+
+    //This is the word that the computer chose.
+var randomWord = allTheWords.randomElement() ?? ""
+    //This counts the lenght of the word
+var randomWordCount = randomWord.count
+
+    //prints the word so I can see it and I can remember will delete before final submission
```

Comment:
Please remember to remove commented out code before submitting, furthermore, you gave yourself instructions to delete the variable "randomWord" but you did not remove it. Things to keep in mind for your next project :)